examples/from-conf/main.go

```diff
@@ -25,32 +23,34 @@ func main() {
 	}()
 
 	// Send a few ILP messages.
-	bday, err := time.Parse(dateOnly, "1856-07-10")
+	tradedTs, err := time.Parse(time.RFC3339, "2022-08-06T15:04:05.123456Z")
 	if err != nil {
 		log.Fatal(err)
 	}
 	err = sender.
-		Table("inventors_go").
-		Symbol("born", "Austrian Empire").
-		TimestampColumn("birthdate", bday). // Epoch in micros.
-		Int64Column("id", 0).
-		StringColumn("name", "Nicola Tesla").
-		At(ctx, time.Now()) // Epoch in nanos.
+		Table("trades_go").
```

Comment:
There are other examples in this dir which should be updated.

Reply:
I updated this one since this one is the example picked up by the cloud console. 
but I can update the other examples too, that won't do any harm.

Reply:
That would be great. This and other examples are meant to be read by users.

Reply:
done